cpp/src/arrow/compute/kernels/scalar_arithmetic.cc

```diff
@@ -820,6 +1081,80 @@ const FunctionDoc pow_checked_doc{
     ("An error is returned when integer to negative integer power is encountered,\n"
      "or integer overflow is encountered."),
     {"base", "exponent"}};
+
+const FunctionDoc sin_doc{"Computes the sine of the elements argument-wise",
```

Comment:
Other docstrings use infinitive / imperative instead of present, such as "Compute the sine of the arguments element-wise".